packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1164,7 +1232,9 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
                   this.groupType,
                   this.userFilters
                 );
-              } catch (_) {}
+              } catch (_) {
+                // no-op
```

Comment:
@gavinbarron probably not the best place to ask this but we don't support adding a logger. However, @sebastienlevert mentioned leveraging the JS SDK on this here https://github.com/microsoftgraph/microsoft-graph-toolkit/issues/1126 and I was wondering, is this something to consider going to v3?

Reply:
Great question @musale Not for v3 as the Observability work is for our v4 JS SDK which is not ready for adoption yet.